tests/src/test/scala/integration/CredentialsIBMPythonCOSTests.scala

```diff
@@ -26,8 +26,9 @@ import org.scalatest.BeforeAndAfterAll
 @RunWith(classOf[JUnitRunner])
 class CredentialsIBMPythonCOSTests extends TestHelpers with WskTestHelpers with BeforeAndAfterAll with WskActorSystem {
```

Comment:
Seems like all these classes with `lazy val defaultKind` can be changed to take a kind variable and remove `RunWith(classOf[JUnitRunner])`.

`class CredentialsIBMPythonCOSTests(kind: String)...`

Reply:
I don't follow
This has being the way we run all our tests multiple times for different kinds, to override certain variables.

The `RunWith(classOf[JUnitRunner])` is required so they get pickup by gradle/junit test we can't remove.
This Classes are not called programmatically so a contructor can't be use.

Reply:
A constructor can still be used when extending the base class.

```
class Base(val kind: String) {} 
class ExtendedBass() extends Test1(kind = "python:3")   {}
```